compose.yml

```diff
@@ -107,6 +107,20 @@ services:
       - DATABASE_PASSWORD=${DATABASE_PASSWORD}
       - DATABASE_USER=${DATABASE_USER}
 
+  postgresql-connection-test:
```

Comment:
This isn't necessary, the Message Status and Notify functions use the exact same way to connect to postgres via psycopg and the 4 connection env vars, please review the container config above for the Message Status function, these are the required env vars.